modules/drivers/lidar/lidar_robosense/component/component_driver.h

```diff
@@ -0,0 +1,97 @@
+/******************************************************************************
+ * Copyright 2021 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+
+#pragma once
+#include <memory>
+#include <string>
+#include <thread>
+
+#include "cyber/cyber.h"
+#include "modules/drivers/lidar/lidar_robosense/driver/driver.h"
+#include "modules/drivers/lidar/lidar_robosense/proto/sensor_suteng.pb.h"
```

Comment:
Code format.

Reply:
anything wrong？

Reply:
Use './apollo.sh format path/to/cpp/header/or/src/files` to format it and you will see.